package/base-files/files/sbin/sysupgrade

```diff
@@ -271,7 +271,7 @@ create_backup_archive() {
 				fi
 			done
 			disabled="$disabled\nexit 0"
-			tar_print_member "/etc/uci-defaults/10_disable_services" "$(echo -e $disabled)" || ret=1
+			tar_print_member "/etc/uci-defaults/10_disable_services" "$(echo -e $disabled)" "$(stat -c "%Y" /etc/rc.d)" || ret=1
```

Comment:
But [`stat` isn’t part of the default busybox configuration](https://github.com/openwrt/openwrt/blob/10180295b817abb3efdc9db3fb4bb62d585fad7a/package/utils/busybox/Config-defaults.in#L805).

Reply:
We don't have `ls -l --time-style="+%s"` so do you have another way to do this?